### PR TITLE
Create /cube/faq page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Create /cube/faq page
- Add `FAQ` in the navigation menu

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/faq
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1tDByDnXmf6QnpvtzxrX3JsgNYuBZbv1Bx7o6mGnbzSg/edit#heading=h.g1eby6bz6419)

## Issue / Card

Fixes [4354](https://github.com/canonical-web-and-design/web-squad/issues/4354)

## Screenshots

image

### CODE REVIEW

File: templates/cube/faq.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <p>The <a href="/cube/study">CUBE Study Materials</a> are available free of charge, and offer a brief overview of some of the topics covered in the CUBE exams.</p>
+      <p>The CUBE Study Labs are available for a small fee. They provide access to virtual Ubuntu terminals and related exercises that simulate the experience of taking an exam.</p>
+      <h3 id="preparatory-materials">Will the preparatory materials train me in using Ubuntu?</h3>
+      <p>No. The Study Materials and Study Labs are meant to refresh your existing knowledge. They are not designed to turn an inexperienced Ubuntu user into a professional.</p>
+      <h3 id="preparatory-materials-coverage">Do the preparatory materials cover everything in the exams?</h3>
+      <p>No. The exams explore a wide range of topics related to their particular subject areas. The preparatory materials cover many of these topics, but not all of them.</p>
+      <p>To see what kinds of topics each microcert covers, consult the “Topics” column on the <a href="/cube/microcerts">Microcertifications</a> page.</p>
+      <h2>Badges</h2>
+      <h3 id="what-are-badges">What are badges?</h3 id="what-are-badges">
```
Comment (carkod) [inline templates/cube/faq.html:47]: Oops copy paste error? `id` added to the closing tag

**Review by carkod — APPROVED** [review]
One typo. Else LGTM

File: templates/cube/faq.html (hunk 1 above)
Comment (minkyngkm) [reply templates/cube/faq.html]: Fixed! 👍

**Comment by anasereijo** [comment]
Hi @minkyngkm this is looking good. Just one thing - the `hr` should be full width

**Comment by minkyngkm** [comment]
> Hi @minkyngkm this is looking good. Just one thing - the `hr` should be full width

Hi @anasereijo updated the `hr` to the full width! Hopefully it looks okay!

**Comment by anasereijo** [comment]
thanks @minkyngkm for making the changes 👍

**Pull request merged**